islandora_checksum_checker.module

```diff
@@ -213,10 +214,22 @@ function islandora_checksum_checker_validate_checksum($pid) {
 
   // Force the cron to run as user 1 so we don't get locked out of objects.
   $user = user_load(1);
+
   drupal_static_reset('islandora_get_tuque_connection');
-  $tuque = islandora_get_tuque_connection($user);
-  $islandora_object = islandora_object_load($pid);
+  try {
+    if (!$tuque = islandora_get_tuque_connection($user)) {
```

Comment:
```islandora_object_load```used at line 232 actually calls internally islandora_get_tuque_connection for the current user, whatever that one is in this context. But you are also calling islandora_get_tuque_connection() directly at line 220.  So i'm not clear about the A) perfomance of opening 2x tuque connections + B) loading the object with the actual "global user" == real logged in one and then making the actual ::getDatastream with the user "1". Is that actually working in an environment with restricted access for objects?